src/msg/async/dpdk/dpdk_rte.cc

```diff
@@ -123,6 +125,21 @@ namespace dpdk {
         args.push_back(string2vector("--no-huge"));
       }
 
+      if (!cct->_conf->ms_dpdk_port_options.empty()) {
```

Comment:
i don't think this compile. please just loop in the `get_str_map(cct->_conf->ms_dpdk_port_options)` without checking it.

Reply:
OK